.cci.jenkinsfile

```diff
@@ -72,7 +72,10 @@ parallel vmcheck: {
         // really, we just need test deps, but meh...
         shwrap("ci/installdeps.sh")
       }
-      stage("Run") {
+      stage("Kola") {
+        fcosKola(cosaDir: "${env.WORKSPACE}", extraArgs: "ext.*")
```

Comment:
```suggestion
        fcosKola(cosaDir: "${env.WORKSPACE}", extraArgs: "ext.*", parallel: nhosts)
```

Reply:
Thanks, folded that in!

Reply:
Will repush after https://github.com/coreos/coreos-ci-lib/pull/21 lands to retest.